src/components/ec-checkbox/ec-checkbox.spec.js

```diff
@@ -204,6 +204,26 @@ describe('EcCheckbox', () => {
       );
       expect(wrapper.element).toMatchSnapshot();
     });
+    it('should render with custom attributes', () => {
+      const wrapper = mountCheckbox({}, {
+        attrs: {
+          'data-test': 'my-data-test',
+          class: 'my-class',
+          id: 'test-id',
```

Comment:
This id is overridden and not visible in the final snapshot. I'd keep it because it's a nice test that the user can't override it but I'd like to add another attribute that doesn't get overridden, e.g. `role="checkbox"` so we can see in the snapshot where custom attributes go in the rendered HTML.